### PR TITLE
Add MDN links to tests (#973)

### PR DESCRIPTION
**Pull request by sarahkadar**
Implements #973 for all tabs. I asked someone at Mozilla (not like the person actually responsible for branding at Mozilla, just someone who happens to work there), and he said, that based on his reading of the [Mozilla branding guide](https://www.mozilla.org/en-US/styleguide/identity/mozilla/branding/), our usage of the logo is OK here, and I agree with him.

### CODE REVIEW

**Comment by sarahkadar** [comment]
also,  it looks a lot better now with the logo.
screenshot 2017-04-01 17 13 51

File: build.js [hunk 1]
```diff
@@ -513,7 +513,19 @@ function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
     else interpolateResults(t.res);
 
     var id = escapeTestName(t.name);
-    var name = t.spec ? ('<a href="' + t.spec + '">' + t.name + '</a>') : t.name;
+    var name = t.name;
+    if (t.spec) {
+        name = '<a href="' + t.spec + '">' + t.name + '</a>';
+        if (t.mdn) {
+            name += ' <a href="' + t.mdn + '"><img src="/mdn.png" width="15" height="13" /></a>&nbsp;';
```
Comment (ljharb) [inline build.js]: Let's please add an "alt" to the image and a "title" to the link, for a11y

File: build.js [hunk 2]
```diff
@@ -513,7 +513,19 @@ function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
...
+        }
+    }
+
+    if (!t.spec && t.mdn) {
+        name += ' <a href="' + t.mdn + '"><img src="/mdn.png" width="15" height="13" /></a>&nbsp;';
```
Comment (ljharb) [inline build.js]: Could the link html be stored in only one place, or perhaps better, a function that takes an href? (This one needs a11y too)

File: build.js [hunk 3]
```diff
@@ -605,6 +617,9 @@ function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
     if ("subtests" in t) {
       t.subtests.forEach(function(subtest) {
         var subtestName = subtest.name;
+        if (subtest.mdn) {
+          subtestName += ' <a href="' + subtest.mdn + '"><img src="/mdn.png" width="15" height="13" /></a>&nbsp;';
```
Comment (ljharb) [inline build.js]: a11y

**Review by ljharb — CHANGES_REQUESTED** [review]

**Comment by sarahkadar** [comment]
@ljharb I've made the changes.

**Review by ljharb — APPROVED** [review]
LGTM

**Comment by zloirock** [comment]
LGTM

**Pull request merged**